ansible/roles/baremetal-ctrl/meta/main.yaml

```diff
@@ -0,0 +1,6 @@
+---
+dependencies:
```

Comment:
Well, these are just package dependencies, they shouldn't be interfering with the execution to the point that we would want to avoid them while debugging. But I get what you mean, I'll keep it in mind.